dask_planner/src/sql/exceptions.rs

```diff
@@ -1,9 +1,10 @@
 use datafusion::error::DataFusionError;
 use pyo3::{create_exception, PyErr};
+use std::fmt::Debug;
 
 create_exception!(rust, ParsingException, pyo3::exceptions::PyException);
 
-pub fn py_type_err(e: DataFusionError) -> PyErr {
+pub fn py_type_err(e: impl Debug) -> PyErr {
```

Comment:
I'm not sure if this is the best pattern to use. Sometimes we want to pass a string and sometimes we want to pass a `DataFusionError` and they both implement the `Debug` trait. Perhaps it would be clearer to have separate functions for each?